[2024-02-01 11:48:48,381] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T11:48:47.339152+00:00 [queued]>
[2024-02-01 11:48:48,388] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T11:48:47.339152+00:00 [queued]>
[2024-02-01 11:48:48,388] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 11:48:48,388] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-01 11:48:48,389] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 11:48:48,398] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_ddl_postgres> on 2024-02-01 11:48:47.339152+00:00
[2024-02-01 11:48:48,401] {standard_task_runner.py:52} INFO - Started process 744 to run task
[2024-02-01 11:48:48,404] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'create_ddl_postgres', 'manual__2024-02-01T11:48:47.339152+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpbersn0_q', '--error-file', '/tmp/tmpm8x442on']
[2024-02-01 11:48:48,405] {standard_task_runner.py:80} INFO - Job 231: Subtask create_ddl_postgres
[2024-02-01 11:48:48,443] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T11:48:47.339152+00:00 [running]> on host 0cb2a3b23a40
[2024-02-01 11:48:48,493] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=create_ddl_postgres
AIRFLOW_CTX_EXECUTION_DATE=2024-02-01T11:48:47.339152+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-01T11:48:47.339152+00:00
[2024-02-01 11:48:51,455] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-01 11:48:54,451] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-01 11:49:08,595] {local_task_job.py:221} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-02-01 11:49:08,598] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 744. PIDs of all processes in the group: [744]
[2024-02-01 11:49:08,598] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 744
[2024-02-01 11:50:08,623] {process_utils.py:143} WARNING - process psutil.Process(pid=744, name='airflow task ru', status='sleeping', started='11:48:48') did not respond to SIGTERM. Trying SIGKILL
[2024-02-01 11:50:08,625] {process_utils.py:80} INFO - Sending the signal Signals.SIGKILL to group 744
[2024-02-01 11:50:08,633] {process_utils.py:75} INFO - Process psutil.Process(pid=744, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='11:48:48') (744) terminated with exit code Negsignal.SIGKILL
[2024-02-01 11:50:08,634] {standard_task_runner.py:162} ERROR - Job 231 was killed before it finished (likely due to running out of memory)
